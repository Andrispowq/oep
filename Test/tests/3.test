m
x 10
z 10
s 10