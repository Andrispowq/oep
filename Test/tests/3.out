Round 0
Oxygene (Thickness: 8.5
OzoneLayer (Thickness: 9.5
CO2Layer (Thickness: 11.5
Oxygene (Thickness: 0.5
Round 0
Oxygene (Thickness: 7.225
OzoneLayer (Thickness: 9.025
CO2Layer (Thickness: 12.775
Round 0
Oxygene (Thickness: 6.141249999999999
OzoneLayer (Thickness: 8.57375
CO2Layer (Thickness: 13.85875
Round 0
Oxygene (Thickness: 5.220062499999999
OzoneLayer (Thickness: 8.1450625
CO2Layer (Thickness: 14.7799375
Round 0
Oxygene (Thickness: 4.437053124999999
OzoneLayer (Thickness: 7.737809374999999
CO2Layer (Thickness: 15.562946875000002
Round 0
Oxygene (Thickness: 3.7714951562499994
OzoneLayer (Thickness: 7.3509189062499996
CO2Layer (Thickness: 16.228504843750002
Round 0
Oxygene (Thickness: 3.2057708828125
OzoneLayer (Thickness: 6.9833729609374995
CO2Layer (Thickness: 16.794229117187502
Round 0
Oxygene (Thickness: 2.724905250390625
OzoneLayer (Thickness: 6.634204312890624
CO2Layer (Thickness: 17.275094749609377
Round 0
Oxygene (Thickness: 2.3161694628320313
OzoneLayer (Thickness: 6.302494097246093
CO2Layer (Thickness: 17.68383053716797
Round 0
Oxygene (Thickness: 1.9687440434072265
OzoneLayer (Thickness: 5.987369392383789
CO2Layer (Thickness: 18.031255956592776
Round 0
Oxygene (Thickness: 1.6734324368961426
OzoneLayer (Thickness: 5.6880009227646
CO2Layer (Thickness: 18.32656756310386
Round 0
Oxygene (Thickness: 1.4224175713617213
OzoneLayer (Thickness: 5.40360087662637
CO2Layer (Thickness: 18.577582428638284
Round 0
Oxygene (Thickness: 1.2090549356574631
OzoneLayer (Thickness: 5.133420832795051
CO2Layer (Thickness: 18.79094506434254
Round 0
Oxygene (Thickness: 1.0276966953088436
OzoneLayer (Thickness: 4.876749791155298
CO2Layer (Thickness: 18.97230330469116
Round 0
Oxygene (Thickness: 0.8735421910125171
OzoneLayer (Thickness: 4.632912301597533
CO2Layer (Thickness: 19.126457808987485
Round 0
Oxygene (Thickness: 0.7425108623606396
OzoneLayer (Thickness: 4.401266686517657
CO2Layer (Thickness: 19.257489137639364
Round 0
Oxygene (Thickness: 0.6311342330065436
OzoneLayer (Thickness: 4.1812033521917735
CO2Layer (Thickness: 19.368865766993462
Round 0
Oxygene (Thickness: 0.536464098055562
OzoneLayer (Thickness: 3.972143184582185
CO2Layer (Thickness: 19.463535901944443
Round 0
OzoneLayer (Thickness: 3.7735360253530756
CO2Layer (Thickness: 19.544005516652778